backend/cmm_helpers.mli

```diff
@@ -1197,58 +1178,31 @@ val unboxed_int64_or_nativeint_array_set :
   Debuginfo.t ->
   expression
 
-(** {2 Getters and setters for unboxed int and float32 fields of mixed
-    blocks} *)
-
-(** The argument structure for getters is parallel to [get_field_computed]. *)
+(** {2 Getters and setters for unboxed fields of mixed blocks}
 
-val get_field_unboxed_int32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
+    The first argument is the heap block to modify a field of.
+    The [index_in_words] should be an untagged integer.
 
-val get_field_unboxed_float32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
+    In constrast to [setfield] and [setfield_computed], [immediate_or_pointer] is not
+    needed as the layout is implied from the name, and [initialization_or_assignment] is
```

Comment:
the layout is not implied from the name anymore, but rather by the `memory_chunk` given as argument